File: nmproducts/src/main/java/com/nm/products/tvseries/model/Episode.java
package com.nm.products.tvseries.model;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class Episode {

	@JsonProperty("Title")
	private String title;
	
	@JsonProperty("Released")
	private String released;
	
	@JsonProperty("Episode")
	private String episode;
	
	@JsonProperty("imdbRating")
	private String imdbRating;
	
	@JsonProperty("imdbID")
	private String imdbID;

	
	@JsonProperty("Title")
	public String getTitle() {
		return title;
	}

	@JsonProperty("Title")
	public void setTitle(String title) {
		this.title = title;
	}

	@JsonProperty("Released")
	public String getReleased() {
		return released;
	}

	@JsonProperty("Released")
	public void setReleased(String released) {
		this.released = released;
	}

	@JsonProperty("Episode")
	public String getEpisode() {
		return episode;
	}

	@JsonProperty("Episode")
	public void setEpisode(String episode) {
		this.episode = episode;
	}

	@JsonProperty("imdbRating")
	public String getImdbRating() {
		return imdbRating;
	}

	@JsonProperty("imdbRating")
	public void setImdbRating(String imdbRating) {
		this.imdbRating = imdbRating;
	}

	@JsonProperty("imdbID")
	public String getImdbID() {
		return imdbID;
	}

	@JsonProperty("imdbID")
	public void setImdbID(String imdbID) {
		this.imdbID = imdbID;
	}
	
}

File: NMMovieWeb/src/app/model/Episode.ts
export class Episode {
  private title: string;

  private released: string;

  private episode: string;

  private imdbRating: string;

  private imdbID: string;

  public getTitle(): string {
    return this.title;
  }

  public setTitle(title: string) {
    this.title = title;
  }

  public getReleased(): string {
    return this.released;
  }

  public setReleased(released: string) {
    this.released = released;
  }

  public getEpisode(): string {
    return this.episode;
  }

  public setEpisode(episode: string) {
    this.episode = episode;
  }

  public getImdbRating(): string {
    return this.imdbRating;
  }

  public setImdbRating(imdbRating: string) {
    this.imdbRating = imdbRating;
  }

  public getImdbID(): string {
    return this.imdbID;
  }

  public setImdbID(imdbID: string) {
    this.imdbID = imdbID;
  }

}
